lib/tests/test_conflicts.rs

```diff
@@ -839,6 +839,79 @@ fn test_update_conflict_from_content_modify_delete() {
     );
 }
 
+#[test]
+fn test_update_conflict_from_content_simplified_conflict() {
+    let test_repo = TestRepo::init();
+    let store = test_repo.repo.store();
+
+    let path = RepoPath::from_internal_string("dir/file");
+    let base_file_id = testutils::write_file(store, path, "line 1\nline 2\nline 3\n");
+    let left_file_id = testutils::write_file(store, path, "left 1\nline 2\nleft 3\n");
+    let right_file_id = testutils::write_file(store, path, "right 1\nline 2\nright 3\n");
+    // Conflict: left - base + base - base + right
+    let conflict = Merge::from_removes_adds(
+        vec![Some(base_file_id.clone()), Some(base_file_id.clone())],
+        vec![
+            Some(left_file_id.clone()),
+            Some(base_file_id.clone()),
+            Some(right_file_id.clone()),
+        ],
+    );
+    let simplified_conflict = conflict.clone().simplify();
+
+    // If the content is unchanged compared to the materialized value, we get the
+    // old conflict id back. Both the simplified and unsimplified materialized
+    // conflicts should return the old conflict id.
+    let materialized = materialize_conflict_string(store, path, &conflict);
+    let materialized_simplified = materialize_conflict_string(store, path, &simplified_conflict);
```

Comment:
Minor: it might help to print what these materialized values are with `assert_snapshot!`